### PR TITLE
update xtype ref to MOC-2.0 and add some MIN/MAX clarification

### PR DESCRIPTION
**Pull request by pdowler**
This PR also now includes definition/explanation of min/max for various xtypes (related to VOTable).

I've tried to be careful here to not specify beyond what we currently do as there is risk to specify further things we haven't implemented. There's also a bit of forward reference to SODA as an example that I'm not really happy about making, but I think in DALI that's OK because it's goal is commonality and usually is pulled up from other specs.

### CODE REVIEW

File: DALI.tex [hunk 1]
```diff
@@ -762,6 +768,9 @@ \subsubsection{Point}
 In spherical coordinates, all longitude values must fall within [0,360] and all 
 latitude values within [-90,90].
 
+There is no general purpose definition of minimum and/or maximum point values, but
+specific services may define something that is applicable in a more limited context.
```
[the remark this answers is not in the document]
Comment (pdowler) [reply DALI.tex:772]: The difficulty is that one can have scalar min/max or min/.max values that are the same type as the param (eg xtype="point" here)... ideally, you'd rather convey sky coverage ~ max as a general region on the sky (something like polygon or moc) and not "the biggest point. 

Maybe the "min point" and "max point" is sort of like a coordinate range, but we don't have any use like that so I'd rather stay clear for now. The other uses of max come from datalink service descriptors describing soda params, so they have some existing use we're trying to describe here.

**Review by jd-au — APPROVED** [review]

File: DALI.tex [hunk 2]
```diff
@@ -777,6 +786,20 @@ \subsubsection{Circle}
 In spherical coordinates, all longitude values must fall within [0,360] and all 
 latitude values within [-90,90].
 
+Circle-valued service parameters may include additional metadata like minimum and
+or maximum value. These are specified using the a custom interpretation of the
```
[the remark this answers is not in the document]
Comment (pdowler) [reply DALI.tex]: fixed

File: DALI.tex [hunk 3]
```diff
@@ -794,6 +817,19 @@ \subsubsection{Polygon}
 winding direction is counter-clockwise (when viewed from the origin toward the 
 sky) as described in \citep{2007ivoa.spec.1030R}.
 
+Polygon-valued service parameters may include additional metadata to describe minimum
+and/or maximum values. These are specified using the a custom interpretation of the
```
[the remark this answers is not in the document]
Comment (pdowler) [reply DALI.tex]: also fixed :-)

File: DALI.tex [hunk 4]
```diff
@@ -841,6 +880,22 @@ \subsubsection{Shape}
 
 The interpretation and constraints on the coordinate values are as specified for the individual xtypes above.
 
+Shape-valued service parameters may include additional metadata to describe minimum
+and/or maximum values. These are specified using the a custom interpretation of the
```
[the remark this answers is not in the document]
Comment (pdowler) [reply DALI.tex]: and again, cut&paste :-)

**Comment by pdowler** [comment]
thnx James; will wait for other editors to have a look at this text.

**Pull request merged**